### PR TITLE
fix service to use as httpOutput when falcosidekick is enabled

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Issif <issif_github@gadz.org>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

> /kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

**What this PR does / why we need it**:

When `falcosidekick` is enabled, its service is not correctly set for `httpOutput` of `falco`

**Which issue(s) this PR fixes**:

https://github.com/falcosecurity/falcosidekick/issues/178

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: falco/.gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 requirements.lock
-Chart.lock
\ No newline at end of file
+Chart.lock
+charts
+charts
```
Comment (cpanato) [inline falco/.gitignore:4]: nit: new line

**Review by cpanato — DISMISSED** [review]
/lgtm

nonblocking comment

**Comment by cpanato** [comment]
@Issif looks like a fix I did break the ci, can you update that in this PR?
in the ct.yaml you need to add the s after the 800

**Comment by leogr** [comment]
> @Issif looks like a fix I did break the ci, can you update that in this PR?
> in the ct.yaml you need to add the s after the 800

I've added a commit to reverting yours. Let's see if CI will not complain :smiley_cat:

File: falco/.gitignore (hunk 1 above)
Comment (leogr) [reply falco/.gitignore:3]: Done by https://github.com/falcosecurity/charts/pull/178/commits/07c0a8470805f78c742f248b93f8ccb858545f3a

**Review by leogr — APPROVED** [review]

**Pull request merged**